File: 54/uname.go
package main

import (
    "fmt"
    "time"
)

func main() {
    fmt.Println("Enter your name")

    var uname string

    fmt.Scanln(&uname)

    weekday := time.Now().Weekday()

    fmt.Printf("Hello %v! Happy, %v", uname, weekday)
}

File: 54/uname.py
#!/bin/python3

from datetime import datetime

name = input("Enter your name: ")

dt = datetime.now()
daynumber = dt.weekday()
week = ("Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday")

print(f"Hello, {name}! Happy {week[daynumber]}!")

